### DISCUSSION TITLE
Display result as fraction

### DISCUSSION THREAD

**Discussion by T1mVo** [sharkdp/numbat#626] [Q&A]
Is it possible to display the results as fractions?

**Comment by Goju-Ryu** [comment]
This is pretty similar to #219 and related issues. Currently it is not possible, but it seems to be a popular ask to have control over result notation style.

**Comment by Goju-Ryu** [comment]
I have begun working on some utility functions to provide this and formatting options for values shown as strings. I'll update you when I’m done.
